### PR TITLE
Setup localization pipeline

### PR DESCRIPTION
**Pull request by apoorvdeshmukh**
This commit focuses on adding and experimenting with configuration files for localization pipeline.

### CODE REVIEW

File: internal/translations/LocProject.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+    "Projects": [
+        {
+            "LanguangeSet": "AzureKatal_Languages",
```
Comment (shueybubbles) [inline internal/translations/LocProject.json:4]: >guange [](http://example.com/codeflow?start=16&length=6)

`LanguageSet` ?

**Review by shueybubbles — CHANGES_REQUESTED** [review]
:clock1:

File: internal/translations/LocProject.json (hunk 1 above)
Comment (apoorvdeshmukh) [reply internal/translations/LocProject.json]: Corrected the typo.

**Comment by apoorvdeshmukh** [comment]
I am experimenting with generating the localization artifacts with [this pipeline](https://mssqltools.visualstudio.com/CrossPlatBuildScripts/_build?definitionId=615). Will mark this ready once the pipeline is successful and yaml for this pipeline is moved to our repo. Currently it is failing with "'Allow scripts to access the OAuth token' not enabled in pipeline" error.

**Pull request closed without merging**